src/components/Renderer/widgets/ArrayWidget.tsx

```diff
@@ -1,9 +1,9 @@
 import * as React from 'react';
 import map from 'lodash/map';
 import get from 'lodash/get';
-import { Flex } from '../../../components/Flex';
+import { Flex } from '../../Flex';
 import { Widget, WidgetProps, getArrayItems } from './widget-util';
-import { RenditionJsonSchemaRenderer } from '../index';
+import { Renderer } from '../index';
```

Comment:
This change actually has the effect of adding an `ErrorBoundary` at every level of the Renderer (as `Renderer` includes an `ErrorBoundary` but `RenditionJsonSchemaRenderer` did not). Whilst it might be considered useful in some cases to catch errors at any level down while rendering a nested object, it's probably overkill and inefficient for normal use.